### PR TITLE
chore: add .npmrc to project

### PR DESCRIPTION
**Pull request by adam2k**
Fix CVE-2022-2564

**To reproduce this PR run the command:**

```
$ yarn add mongoose@^6.4.7
```

### CODE REVIEW

**Review by Uzlopak — CHANGES_REQUESTED** [review]
Nah. I would prefer to create a .npmrc with package-lock=false and deleting the package-lock. Then everybody can upgrade mongoose to latest.

**Comment by adam2k** [comment]
> Nah. I would prefer to create a .npmrc with package-lock=false and deleting the package-lock. Then everybody can upgrade mongoose to latest.

That sounds like a good idea as long as there aren't any other dependencies in the future.  I can apply the changes to this PR.

**Comment by Uzlopak** [comment]
Yes please revert the change in package.json and do the change i recommended. I would merge it then...

**Comment by adam2k** [comment]
@Uzlopak I'm noticing that `.npmrc` is in the `.gitignore` from this [commit](https://github.com/flash-oss/medici/commit/2339876b944657bf7e55cc515e1a5a8df70357c0).  I'll need to remove that line as well for the proposed change.

**Comment by Uzlopak** [comment]
Oh, please adapt the workflow accordingly

**Comment by adam2k** [comment]
@Uzlopak the errors in the Check Code CI job [here](https://github.com/flash-oss/medici/runs/7697598699?check_suite_focus=true).  Looks to be related to this open PR https://github.com/flash-oss/medici/pull/73

**Comment by adam2k** [comment]
In this [CI job](https://github.com/flash-oss/medici/runs/7697598776?check_suite_focus=true) the failure is due to the suggestion of removing `package-lock.json`.  We'll either need `package-lock.json` or `npm-shrinkwrap.json` in place for the CI to pass.

This change might be better with the original `package-lock.json` upgrade path for `mongoose`.  @Uzlopak how would you like to proceed?

**Comment by adam2k** [comment]
Ok, I think I have an idea.  I'll edit the `ci.yml` and see if it'll work.

**Comment by Uzlopak** [comment]
Let me start my laptop...

**Comment by adam2k** [comment]
Thanks!  I saw your PR went in for https://github.com/flash-oss/medici/pull/75.  I'll rebase and try this out again.

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock=true
```
Comment (Uzlopak) [inline .npmrc:1]:
```suggestion
package-lock=false
```

**Comment by Uzlopak** [comment]
rebase please

**Pull request merged**